docs/router.md

```diff
@@ -35,10 +35,18 @@ Creating a route with path-based matching supports: exact match: `foo`; prefix m
 $ rio route add $name/path to $target
 
 # to add prefix
-$ rio route add $name/path* to $target
+$ rio route add $name/path'*' to $target
```

Comment:
Looks like this was just outdated from before ? Didn't realize syntax changed.

Reply:
Doesn't work with bash cause you have to escape *.